index.html

```diff
@@ -2,12 +2,75 @@
 <html>
 <head>
 <link href="style.css" rel="stylesheet" type="text/css">
+<script src="https://kit.fontawesome.com/173db1d6d6.js" crossorigin="anonymous"></script>
 <title>Portafolio</title>
 </head>
 <body>
 
-<h1>This is a Heading</h1>
-<p>This is a paragraph.</p>
+    
+<div class="wrapper">
+
+    <nav class="toolbar">
+        <ul>
+            <li class="item">Edahí Gutiérrez </li>        
+            <li id="item-right" class="item">&#9776;</li>                    
+        </ul>        
+    </nav>
```

Comment:
- As part of the project requirements, you are to wrap your `nav` tag around a `header` tag. This will help improve the semantics of your website and improve the accessibility of your site 🙏

Reply:
Thanks I agree